modules/cvtt/register_types.h

```diff
@@ -29,6 +29,13 @@
 /*************************************************************************/
 
 #ifdef TOOLS_ENABLED
+
+#ifndef CVTT_REGISTER_TYPES_H
+#define CVTT_REGISTER_TYPES_H
+
 void register_cvtt_types();
 void unregister_cvtt_types();
-#endif
+
+#endif // CVTT_REGISTER_TYPES_H
+
+#endif // TOOLS_ENABLED
```

Comment:
All your edits remove the newline at the end of file, you should check the configuration of your IDE.
Text files should end with a newline as per the POSIX standard.